tests/specs/verification.k

```diff
@@ -196,8 +243,24 @@ module VERIFICATION
   rule X /Int Y >=Int N:Int => X >=Int N *Int Y
     requires Y >Int 0 [simplification, concrete(N, Y)]
 
+  rule X >Int Y *Int Z => ((X -Int 1) /Int Z) +Int 1 >Int Y
+    requires Z >Int 0
+     andBool X >=Int 1 [simplification, concrete(X, Z)]
+
+  rule 0 >Int Y *Int Z => 0 >Int Y
+    requires Z >Int 0 [simplification, concrete(Z)]
+
+  rule X *Int Y >=Int Z => X >=Int ((Z -Int 1) /Int Y) +Int 1
+    requires Y >Int 0 
+     andBool Z >=Int 1 [simplification, concrete(Z, Y)]
+
+  rule X *Int Y >=Int 0 => X >=Int 0
+    requires Y >Int 0 [simplification, concrete(Y)]
+
```

Comment:
Changes included from pact-verification-dev: I need to double-check these and see if there are any dual cases to be added for completeness